code/modules/mob/living/carbon/human/species_types/mushpeople.dm

```diff
@@ -0,0 +1,46 @@
+/datum/species/mush //mush mush codecuck
+	name = "Mushroomperson"
+	id = "mush"
+	mutant_bodyparts = list("caps", "tail_human", "ears", "wings")
+	default_features = list("caps" = "Round", "tail_human" = "None", "ears" = "None", "wings" = "None")
```

Comment:
caps won't show up if anyone has an idea on why that is please tell me!

Reply:
why do they have access to ears and tails?

Reply:
lol uuh